packages/openapi-generator/src/file-serializer/schema.spec.ts

```diff
@@ -0,0 +1,173 @@
+import { serializeSchema } from './schema';
```

Comment:
I know the readme.ts is not touched but could we have a follow up that we use some shared content in both generators and consider the `COMMON-README-PART.md` as a starting point? We should than also generate this shared file via code and check each link in our E2E test that it is at least still present.

Reply:
We already have tickets for this, I improved the descriptions.